docs/types_distributedExecution.js.html

```
@@ -0,0 +1,256 @@
+  const pubkeys = Array.isArray(validatorPubkeys) ? validatorPubkeys : [];
+  const thr = Math.max(1, Number(threshold) || 1);
+  let valid = 0;
+  for (const pubkey of pubkeys) {
+    if (typeof pubkey !== 'string' || !pubkey) continue;
+    const sigHex = witness.signatures[pubkey];
+    if (!sigHex || typeof sigHex !== 'string') continue;
+    try {
+      const k = new Key({ pubkey });
+      const sig = Buffer.from(sigHex, 'hex');
+      if (k.verifySchnorr(messageBuffer, sig)) valid++;
+      if (valid >= thr) return true;
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)